### PR TITLE
Test empty image

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
I tried to create a dummy Numpy product for the `empty` instrument, but I am not entirely sure I did in the correct way, hence the tag.

This will be helpful for dealing with [https://github.com/oda-hub/oda_api/issues/60](https://github.com/oda-hub/oda_api/issues/60)

File: tests/test_server_basic.py [hunk 1]
```diff
@@ -1009,3 +1009,45 @@ def test_example_config(dispatcher_test_conf):
     print("\n\n\ntest_config_keys", test_config_keys)
 
     assert set(example_config_keys) == set(test_config_keys)
+
+
+def test_mosaic_image(dispatcher_live_fixture):
```
Comment (volodymyrss) [inline tests/test_server_basic.py:1014]: it's a bit strange that "mosaic" is mentioned in generic dispatcher. It has concept of Image but not details like mosaic. But it's minor.

Comment (burnout87) [reply tests/test_server_basic.py:1014]: I am not too familiar with these concepts and the relative terminology.

So just using image would be enough? Whreas mosaic is a more instrument-specific terminology?

Comment (volodymyrss) [reply tests/test_server_basic.py:1014]: It depends. But what is relevant here is what is already present in dispatcher. You can see that there is `ImageQuery` and `ImageProduct` but no mosaics. It's a reasonable choice to introduce image but not mosaic.

**Comment by volodymyrss** [comment]
It's related to some issue, yes?

**Comment by burnout87** [comment]
> It's related to some issue, yes?

In order to tackle what we discussed here [https://github.com/oda-hub/oda_api/issues/60](https://github.com/oda-hub/oda_api/issues/60), I wanted to have one of the empty/dummy instruments to provide as a product an image.

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**